src/platform/storage_winuser.c

```diff
@@ -57,6 +57,14 @@ RegNotifyChangeKeyValue(
     _In_opt_ HANDLE hEvent,
     _In_ BOOL fAsynchronous
     );
+
+WINADVAPI
```

Comment:
nit: Given the slowly increasing number of "Restricted build" declarations, should we consider grouping them in a header made specifically for this purpose?
It would keep them out of the way for the 99% of the time we are not concerned with them :)